### PR TITLE
fix(web/appeals): semantic-release (boas-1518)

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
## Describe your changes

removed the following packages as they where not used anymore

- semantic-release
- multi-semantic-release
- @semantic-release/exec

removed release pipeline as it's not needed anymore

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -5,10 +5,6 @@
 			"outputs": ["src/server/static/scripts/**", "src/server/static/styles/**"],
 			"dependsOn": ["^build", "$NODE_ENV", "$APP_RELEASE"]
 		},
-		"build:release": {
-			"outputs": ["src/server/static/scripts/**", "src/server/static/styles/**"],
-			"dependsOn": ["^build", "$NODE_ENV", "$APP_RELEASE"]
-		},
```
Comment (beejjacobs) [inline turbo.json:11]: This is actually still needed, this is the script used to build the web apps. The CI pipeline runs it, as do the Docker builds.

**Review by beejjacobs — COMMENTED** [review]
Looks good, just need to reinstate the build:release turbo script

File: turbo.json (hunk 1 above)
Comment (francescobrenci-AR) [reply turbo.json]: cool - I've added it back

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**